plugin.video.fanfilm/resources/lib/sources/pl/iitvx.py

```diff
@@ -72,6 +72,12 @@ def work(self, link, testDict):
             valid, host = source_utils.is_host_valid(link, testDict)
             if not valid: return 0
             return host, q, link
+
+        if str(link).startswith("https"):
```

Comment:
Ładnie, ale to nie zadziała. W sensie Twój kod nigdy się nie wykona.
`str(link).startswith("http")` wyłapie też "https". Tam jest `startwith` czyli rozpoczyna się od. Jakby było sprawdzane "http://" to wtedy nie złapie "https".

Zresztą to jest ten sam kod. Proponuję unikać pasty copy (Copy-Paste).
Może zostać jak było (będzie moim zdaniem działać tak samo). Albo pierwsze sprawdzenie można zamienić na bardziej restrykcyjne:
```python
    if str(link).startswith("http://") or str(link).startswith("https://"):
```
Ale to w obecnym kodzie chyba nie ma sensu.
Tak jak mnie zastanawia `return 0`. Zamiast samo `return` lub jawnie `return None`. Nie wydaje się, żeby to musiał być integer, skoro jak jest valid to zwraca tuple. Pewnie mogło by być `None` lub `False`, cokolwiek z czego `bool()` daje false. Ale to już zupełna dygresja, nie zachęcam do zmian :-)

Reply:
Racja, dałem ciała.